### ISSUE TITLE
Content revisions for ITF 21-0966 Form from Veteran Facing Forms

### ISSUE THREAD

**Issue by jeana-adhoc** [department-of-veterans-affairs/va.gov-team#74328]
# Content, accessibility, information architecture (CAIA) new initiative collaboration request

Use this ticket to start collaboration on a new initiative with the sitewide content, accessibility, and information architecture (CAIA) team.

**Note:** If you’re already partnering with us on an initiative, you don't need to fill out this request form. If you don’t have access to your initiative’s CAIA epic, post a message in the [#sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-acessibility-ia) slack channel and tag `@Terry Nichols` and `@Lily Strelich`.

## About your team

- **Team name**: Veteran Facing FOrms
- **OCTO product owner:** Matthew Dingee
- **Product name:**  Submit an Intent to file - VA Form 21-0966
- **Product manager:**  @tbaker1026 
- **Slack channel:** [#veteran-facing-forms](https://join.slack.com/share/enQtNjUxOTU5NjcxODc3MC0yYzRkNmE4Yjg4Njg5ZTNhYjY4NDM4Y2YwZDFmY2M4YmExNmM4MTk4ZmI5MWY1ZDVjYjExYzQ2ZmNmMjAxNTZi)
- **Dedicated content writer on your team (if you have one):** `Insert name and/or GH handle`
- **Dedicated a11y specialist on your team (if you have one):** @jeana-adhoc 
 
### Which of these descriptions best fits the work we’ll partner on?
Select all that apply.
- [ ] Digitizing a new form
- [ ] Creating a new digital tool
- [X] Updating an existing form or tool
- [ ] Translating a form or tool into Spanish
- [ ] Adding new unauthenticated content to VA.gov
- [ ] Updating existing unauthenticated content on VA.gov
- [ ] Something else:  `insert description`

### What's the nature of your initiative and desired outcomes?

We have already worked with @megzehn on content writing/review for our stand alone form 21-0966. Since then, we needed to add a new flow that allows an even easier/faster submission of a form.  Also, during that time, I provided CAIA with a document that outlines inconsistencies we noticed in ITF language across VA.gov. [That work is being done here.](https://github.com/orgs/department-of-veterans-affairs/projects/929/views/1?pane=issue&itemId=49703686)

## Collaboration timeframe

**Note:** We work on nearly every OCTO product and manage all unauthenticated content on VA.gov, so we will need to prioritize intake requests based on overall workload and VA and OCTO priorities.

### Is this work tied to a Congressional mandate, change in law or policy, or upcoming event with a specific deadline?
- [ ] Yes
- [X] No
- [ ] I’m not sure

### Where are you at in your timeline?
Tell us briefly about what you're working on now (such as initial discovery, wireframing, or usability research planning) and add any known dates for upcoming milestones or deadlines.

We are close to shipping. We've already gone through staging review, and even with these changes, we won't need to go through staging again. But I need a second set of eyes on this content, and to help guide us on how to make the language more consistent with other Intent to File language across VA.gov

### Which phases of the collaboration cycle have you completed?
Select all that apply.
- [ ] Design intent 
- [ ] Midpoint review 
- [X] Staging 
- [ ] None. This initiative isn’t going through the collaboration cycle.

### Collaboration cycle ticket 

If you’re going through the collaboration cycle, provide your ticket number and link:
- **Ticket Number:** #56287 

## Supporting artifacts
Provide links to any supporting artifacts that can help us better understand your initiative and begin collaboration. Include artifacts like your product outline, user flows, mockups and prototypes, or any draft content.
- [Figma workflow](https://www.figma.com/file/EFJLJ7W4yBbujH1yVD122s/WIP-Migration---21-0966---Intent-to-file?type=design&node-id=8%3A118501&mode=design&t=Eg8UOb4k4dYGhUMi-1)
- [PR For content changes](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/pull/968/files) from Jeana

## Next steps
- [ ] Post a link to this issue in the [#sitewide-content-sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-accessibility-ia) slack channel and tag `@Terry Nichols`. We’ll reply within 24 hours to acknowledge receipt. We'll then review the artifacts in more detail and work with you to determine timing for collaboration. Depending on the work, we may schedule a kickoff meeting to better understand how we'll partner together.
- [ ] Review the remaining next steps in the **VFS team tasks** section here.

## Tasks

<br>

```[tasklist]
### VFS team tasks
- [ ] **If you also need engineering support from the public websites team**, fill out their intake request form as well.<br> [Open a public websites intake request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E)
- [ ] **If you're requesting a page/URL redirect, a URL change, or a vanity URL**, submit an additional request for this work. <br> [Open a redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+kristinoletmuskat%2C+strelichl%2C+FranECross&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&projects=&template=redirect-request.md&title=Redirect+Request)
- [ ] **If you're creating an experimental design**, also contact the design system team. <br> [Read more about experimental designs](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) <br> [Suggest an addition or update to the design system team](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- [ ] **If you're requesting a11y support for usability research**, submit a research support request next. <br> [Open an a11y research support request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name)
- [ ] **If you are requesting content support**, please share your content source of truth (preferably as a SharePoint document).
```

```[tasklist]
### CAIA operations
- [ ] Initial Review - @coforma-terry and @strelichl 
```

```[tasklist]
### CAIA a11y tasks
```

```[tasklist]
### CAIA content tasks
```

```[tasklist]
### CAIA IA tasks
```

**Comment by megzehn** [comment]
Update: Team is aiming to go live with this form the week of February 5-9. I will sync with Jeana on 1/29 and aim to have copy by 1/30.

**Comment by megzehn** [comment]
I reviewed the [updated language that @jeana-adhoc is working on](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/pull/979). Jeana and I talked through a few more tricky content challenges related to third-party representatives who can sign. Jeana is asking the stakeholder a few more follow-up questions.
